src/lib/device-proxy.ts

```diff
@@ -136,6 +136,25 @@ export const proxy = async (req: Request, res: Response) => {
 	}
 };
 
+export async function throttledForEach<T, U>(
```

Comment:
I'd rather this was exported from a `utils` module and received the delay to apply as an argument.

Reply:
Urg, I did mean to make the delay configurable - I'll get that updated

Reply:
done